Assignment-06-Fashionz/styles/css/fashionz.css

```diff
@@ -0,0 +1,181 @@
+body {
+  background-color: rgba(42, 43, 48, 255);
+  padding: 0%;
+  margin: 0;
+}
+
+.parent_container {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: auto;
+  position: relative;
+  top: 60px;
+  margin-bottom: 200px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.brand {
+  height: 60px;
+}
+
+.header_content {
+  height: 90px;
+}
+
+.container {
+  position: relative;
+}
+
+.trigger {
+  background-color: #ffffff;
+  color: #fff;
+  padding: 10px;
+  cursor: pointer;
+  border-radius: 50%;
+  height: 70px;
+  width: 70px;
+  text-align: center;
+  transition: background-color 0.8s;
+}
+
+.trigger:hover {
+  background-color: #cfd0d1;
+}
+
+.dialog-box {
+  display: none;
+  background-color: #ffff;
+  border: 1px solid #ccc;
+  padding-bottom: 30px;
+  padding-right: 80px;
+  position: absolute;
+  left: -230%; /* Adjust this value to control the position to the left */
+  width: 200px;
+  top: 100px;
+  z-index: 1;
+  pointer-events: none;
+}
+
+.container:hover .dialog-box {
+  display: flex;
+  pointer-events: auto;
+  font-family: "Roboto", "sans-serif";
```

Comment:
Let's not repeat font-family if it is common

Reply:
Resolved font-family repeatition